### PR TITLE
Update react => v16

### PR DESCRIPTION
**Pull request by adamhooper**
Uncomment tests

### CODE REVIEW

File: assets/js/WorkflowMetadata.test.js [hunk 1]
```diff
@@ -39,46 +39,43 @@ describe('WorkflowMetadata - private mode', () => {
...
-  //    });
-  //  });
-  //});
+  it('modal operates', (done) => {
+    var publicLink = wrapper.find('.test-button');
+    expect(publicLink).toHaveLength(1);
+    publicLink.first().simulate('click');
+
+    setImmediate(() => {
+      // The insides of the Modal are a "portal", that is, attached to root of DOM, not a child of Wrapper
```
Comment (jstray) [inline assets/js/WorkflowMetadata.test.js]: This comment is now wrong

File: assets/js/WorkflowNavBar.test.js [hunk 2]
```diff
@@ -130,39 +130,31 @@ describe('WorkflowNavBar', () => {
...
-    //  // check that link has rendered correctly
-    //  let linkField = shareModal.find('.test-link-field');
-    //  expect(linkField.length).toBe(1);
-    //  // Need to fix this once correct link string in place
-    //  // expect(linkField.props().placeholder).toEqual("");
-    //
-    //  expect(api.setWorkflowPublic.mock.calls.length).toBe(1);
-    //  done();
-    //});
+    // The insides of the Modal are a "portal", that is, attached to root of DOM, not a child of Wrapper
```
Comment (jstray) [inline assets/js/WorkflowNavBar.test.js]: Again, comment is out of date

File: assets/js/wfparameters/AceEditor.js [hunk 3]
```diff
@@ -25,6 +25,7 @@ export default class WorkbenchAceEditor extends React.Component {
       <div className='parameter-margin'>
         <div className='label-margin t-d-gray content-3'>{this.props.name}</div>
         <AceEditor
+          editorProps={{ $blockScrolling: Infinity }}
```
Comment (jstray) [inline assets/js/wfparameters/AceEditor.js]: What does this do?

Comment (adamhooper) [reply assets/js/wfparameters/AceEditor.js]: Avoids a warning. Commented.

**Pull request merged**